pilot/pkg/networking/plugin/authz/conversion.go

```diff
@@ -0,0 +1,169 @@
+package authz
+
+import (
+	"fmt"
+	"strings"
+
+	envoy_rbac "github.com/envoyproxy/go-control-plane/envoy/config/rbac/v2alpha"
+	envoy_matcher "github.com/envoyproxy/go-control-plane/envoy/type/matcher"
+
+	"istio.io/istio/pilot/pkg/networking/plugin/authz/matcher"
+	"istio.io/istio/pilot/pkg/networking/plugin/authz/rbacfilter"
+	authn_v1alpha1 "istio.io/istio/pilot/pkg/security/authn/v1alpha1"
+	"istio.io/istio/pkg/spiffe"
+)
+
+func permissionForKeyValues(key string, values []string) *envoy_rbac.Permission {
```

Comment:
I know that it is a refactoring, but maybe we can add some docstring here for this function? Same with some other big functions.

Reply:
Done